JDK10 master ut failed

```
Running org.apache.dubbo.rpc.protocol.dubbo.telnet.ListTelnetHandlerTest
Tests run: 6, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.129 sec <<< FAILURE! - in org.apache.dubbo.rpc.protocol.dubbo.telnet.ListTelnetHandlerTest
testList(org.apache.dubbo.rpc.protocol.dubbo.telnet.ListTelnetHandlerTest)  Time elapsed: 0.02 sec  <<< ERROR!
org.apache.dubbo.rpc.RpcException: Fail to start server(url: dubbo://127.0.0.1:20885/demo?channel.readonly.sent=true&codec=dubbo&heartbeat=60000) Failed to bind NettyServer on /127.0.0.1:20885, cause: Address already in use
	at org.apache.dubbo.rpc.protocol.dubbo.telnet.ListTelnetHandlerTest.testList(ListTelnetHandlerTest.java:120)
Caused by: org.apache.dubbo.remoting.RemotingException: Failed to bind NettyServer on /127.0.0.1:20885, cause: Address already in use
	at org.apache.dubbo.rpc.protocol.dubbo.telnet.ListTelnetHandlerTest.testList(ListTelnetHandlerTest.java:120)
Caused by: java.net.BindException: Address already in use
```

https://api.travis-ci.org/v3/job/430398157/log.txt

--- a/dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/ListTelnetHandlerTest.java
+++ b/dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/telnet/ListTelnetHandlerTest.java
@@ -82,7 +82,7 @@ public class ListTelnetHandlerTest {
     public void testListDetailService() throws RemotingException {
         mockInvoker = mock(Invoker.class);
         given(mockInvoker.getInterface()).willReturn(DemoService.class);
-        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:20885/demo"));
+        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:30005/demo"));
         given(mockInvoker.invoke(any(Invocation.class))).willReturn(new RpcResult("ok"));
         mockChannel = mock(Channel.class);
         given(mockChannel.getAttribute("telnet.service")).willReturn("org.apache.dubbo.rpc.protocol.dubbo.support.DemoService");
@@ -97,7 +97,7 @@ public class ListTelnetHandlerTest {
     public void testListService() throws RemotingException {
         mockInvoker = mock(Invoker.class);
         given(mockInvoker.getInterface()).willReturn(DemoService.class);
-        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:20885/demo"));
+        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:30006/demo"));
         given(mockInvoker.invoke(any(Invocation.class))).willReturn(new RpcResult("ok"));
         mockChannel = mock(Channel.class);
         given(mockChannel.getAttribute("telnet.service")).willReturn("org.apache.dubbo.rpc.protocol.dubbo.support.DemoService");
@@ -112,7 +112,7 @@ public class ListTelnetHandlerTest {
     public void testList() throws RemotingException {
         mockInvoker = mock(Invoker.class);
         given(mockInvoker.getInterface()).willReturn(DemoService.class);
-        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:20885/demo"));
+        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:30007/demo"));
         given(mockInvoker.invoke(any(Invocation.class))).willReturn(new RpcResult("ok"));
         mockChannel = mock(Channel.class);
         given(mockChannel.getAttribute("telnet.service")).willReturn(null);
@@ -143,7 +143,7 @@ public class ListTelnetHandlerTest {
     public void testListDefault() throws RemotingException {
         mockInvoker = mock(Invoker.class);
         given(mockInvoker.getInterface()).willReturn(DemoService.class);
-        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:20885/demo"));
+        given(mockInvoker.getUrl()).willReturn(URL.valueOf("dubbo://127.0.0.1:30008/demo"));
         given(mockInvoker.invoke(any(Invocation.class))).willReturn(new RpcResult("ok"));
         mockChannel = mock(Channel.class);
         given(mockChannel.getAttribute("telnet.service")).willReturn("org.apache.dubbo.rpc.protocol.dubbo.support.DemoService");

